public/images/defaultImageURL.js

```diff
@@ -0,0 +1 @@
+export const defaultImageURL = "/images/default-image.png";
```

Comment:
If we are only going to export one thing from here then we might want to consider export default (this will save us a few {} elsewhere. 

Depends where you want to do with this though.